src/accessories/WyzeLight.js

```diff
@@ -72,6 +76,7 @@ module.exports = class WyzeLight extends WyzeAccessory {
   }
 
   async setBrightness(value, callback) {
+    await this.sleep(500);
```

Comment:
I think the sleep would better be placed after the await this.setProperty line

Reply:
Needs to happen before, not after, otherwise the bug will still persist (I tested it). Setting it after means setting brightness and power could still happen simultaneously & out of order.